board/stm32fx/llrtc.h

```diff
@@ -0,0 +1,57 @@
+#define RCC_BDCR_OPTIONS (RCC_BDCR_RTCEN | RCC_BDCR_RTCSEL_0 | RCC_BDCR_LSEON)
+#define RCC_BDCR_MASK (RCC_BDCR_RTCEN | RCC_BDCR_RTCSEL | RCC_BDCR_LSEMOD | RCC_BDCR_LSEBYP | RCC_BDCR_LSEON)
+
+void rtc_init(void){
```

Comment:
How different is this on the H7?

Reply:
At minimum some macros naming is different for H7 and also few isn't available

Reply:
Which ones are you talking about specifically? maybe let's also keep a common version of this with only the (tiny?) differences abstracted out